File: fixtures/scale/csharp/medium/Functions/TrigFunctions.cs
using System;

namespace Calc.Functions
{
    public static class TrigFunctions
    {
        public static void Install(FunctionRegistry registry)
        {
            registry.Define("sin", args => Math.Sin(args[0]));
            registry.Define("cos", args => Math.Cos(args[0]));
            registry.Define("tan", args => Math.Tan(args[0]));
            registry.Define("asin", args => Math.Asin(args[0]));
            registry.Define("acos", args => Math.Acos(args[0]));
            registry.Define("atan", args => Math.Atan(args[0]));
        }
    }
}

File: fixtures/scale/php/medium/src/Functions/TrigFunctions.php
<?php

namespace Calc\Functions;

class TrigFunctions
{
    public static function install(FunctionRegistry $registry): void
    {
        $registry->define('sin', fn ($x) => sin($x));
        $registry->define('cos', fn ($x) => cos($x));
        $registry->define('tan', fn ($x) => tan($x));
        $registry->define('asin', fn ($x) => asin($x));
        $registry->define('acos', fn ($x) => acos($x));
        $registry->define('atan', fn ($x) => atan($x));
    }
}
